specs-go/state.go

```diff
@@ -33,3 +33,19 @@ type State struct {
 	// Annotations are key values associated with the container.
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
+
+type SeccompState struct {
+	// Version is the version of the specification that is supported.
+	Version string `json:"ociVersion"`
+	// SeccompFd is the file descriptor for Seccomp User Notification
+	SeccompFd int `json:"seccompFd"`
+	// Pid is the process ID on which the seccomp filter is applied
+	Pid int `json:"pid"`
+	// PidFd is a pidfd for the process on which the seccomp filter is
+	// applied
```

Comment:
```suggestion
	// PidFd is is the index of the file descriptor in the
	// `SCM_RIGHTS` array referring to the target process file
	// descriptor (e.g as returned by `pidfd_open(2)` or by
	// `clone(2)` with the `CLONE_PID` flag).
```

Reply:
Fixed, thanks! I also updated the description of the `pid` file to include is the pid as seen by the runtime (as config-linux.md says)